internal/core/server/workspace.go

```diff
@@ -704,8 +705,71 @@ func (ws *Workspace) DescribeProcesses(ctx context.Context, input *api.DescribeP
 				ID:       component.ID,
 				Name:     component.Name,
 				Provider: "docker",
-				Running:  state.Running,
 			}
+
+			containerInfo, err := ws.Docker.ContainerInspect(ctx, state.ContainerID)
+			if err == nil {
```

Comment:
In general, we should prefer early returns. If this becomes a separate function like `GetContainerStats`, then you can do `if err != nil { return ...`, which outdents all the code a layer.